apps/consumer-client/src/podExampleConstants.ts

```diff
@@ -35,9 +35,34 @@ export const EXAMPLE_GPC_CONFIG = `{
         },
         "owner": {
           "isRevealed": false,
-          "isOwnerID": true
+          "isOwnerID": true,
+          "isMemberOf": "admissibleOwners"
```

Comment:
This "admissableOwners" pattern is going to be very difficult for someone experimenting in the UI to get right, since owner ID is different for each user, and not displayed in the UI.  I'd suggest using a different entry.